src/components/repository-list-item.component.js

```diff
@@ -1,119 +1,127 @@
+`;
+
+const DescriptionText = ColoredText.extend`
+  font-family: ${styledFonts.fontPrimaryLight};
+`;
+
+const ExtraInfoWrapper = styled.View`
+  flex-direction: row;
+  flex: 1;
+  padding-top: 5;
+`;
+
+const ExtraInfoText = styled.Text`
+  color: ${colors.greyDark};
+  padding-left: 3;
+  padding-top: 2;
+  margin-right: 15;
+  font-family: ${styledFonts.fontPrimary};
+  font-size: ${normalize(10)};
+`;
+
+const ExtraInfoForksText = ExtraInfoText.extend`
+  padding-left: 0;
+  margin-right: 13;
+`;
 
 const renderTitle = (repository, showFullName) => (
-  <View style={styles.wrapper}>
-    <View style={styles.repositoryContainer}>
-      <View style={styles.titleWrapper}>
-        <Text style={styles.title}>
+  <ListItemWrapper>
+    <RepoContainer>
+      <TitleWrapper>
+        <TitleText>
           {showFullName ? repository.full_name : repository.name}
-        </Text>
+        </TitleText>
         {repository.private && (
-          <View style={styles.privateIconContainer}>
+          <PrivateIconContainer>
             <Icon
               size={16}
               name="lock"
               type="octicon"
               color={colors.greyDarkest}
             />
-          </View>
+          </PrivateIconContainer>
         )}
-      </View>
-      <Text style={styles.description}>
-        {emojifyText(repository.description)}
-      </Text>
-    </View>
-    <View style={styles.extraInfo}>
-      <Icon
-        containerStyle={styles.extraInfoIcon}
-        name="star"
-        type="octicon"
-        size={15}
-        color={colors.greyDark}
-      />
+      </TitleWrapper>
+      <DescriptionText>{emojifyText(repository.description)}</DescriptionText>
+    </RepoContainer>
+    <ExtraInfoWrapper>
+      <Icon name="star" type="octicon" size={15} color={colors.greyDark} />
 
-      <Text style={styles.extraInfoSubject}>
+      <ExtraInfoText>
         {abbreviateNumber(repository.stargazers_count)}
-      </Text>
+      </ExtraInfoText>
 
       <Icon
-        containerStyle={styles.extraInfoIcon}
         name="repo-forked"
         type="octicon"
         size={15}
         color={colors.greyDark}
       />
 
-      <Text
-        style={[styles.extraInfoSubject, { paddingLeft: 0, marginRight: 13 }]}
-      >
+      <ExtraInfoForksText>
         {abbreviateNumber(repository.forks_count)}
-      </Text>
+      </ExtraInfoForksText>
 
       {repository.language !== null && (
         <Icon
-          containerStyle={styles.extraInfoIcon}
           name="fiber-manual-record"
           size={15}
           color={languageColors[repository.language]}
         />
       )}
 
-      <Text style={styles.extraInfoSubject}>{repository.language}</Text>
-    </View>
-  </View>
+      <ExtraInfoText>{repository.language}</ExtraInfoText>
+    </ExtraInfoWrapper>
+  </ListItemWrapper>
 );
 
+const styles = StyleSheet.create({
+  title: {
+    color: colors.primaryDark,
+    fontFamily: styledFonts.fontPrimarySemiBold,
+  },
+});
+
```

Comment:
We should be able to get rid of this last one when #619 gets merged.

The `<ListItem>` can be then styled as follow:

```js
const Repository = styled(ListItem).attrs({
  titleStyle: {
    color: colors.primaryDark,
    fontFamily: styledFonts.fontPrimarySemiBold,
  },
})``;
```

the `underlayColor` prop can also be merged there as it's is a pure presentation prop

Reply:
#619 is merged, you should now be able to get rid of this last thingy 🎉